src/cbor/test_decoder.rs

```diff
@@ -396,3 +396,9 @@ fn test_too_large_input() {
     bytes.extend_from_slice(&array);
     test_decoder_error(bytes, CborError::InputTooLarge);
 }
+
+#[test]
+fn test_invalid_input() {
+    let bytes = vec![0xab, 0x65];
```

Comment:
Should we just simplify this to `vec![0x60]` (maybe with a comment) to make it clear we don't support decoding strings?

Reply:
done